react/src/DatePicker/DatePicker.stories.tsx

```diff
@@ -88,3 +88,9 @@ export const CustomInput = Template.bind({})
 CustomInput.args = {
   inputElement: <CustomInputButton />,
 }
+
+export const MobileCustomInput = Template.bind({})
+MobileCustomInput.args = {
+  inputElement: <CustomInputButton />,
```

Comment:
Doesn't look great in mobile view but I guess good enough to see that something gets replaced in mobile view haha